lua/wikis/rematch/Infobox/Person/Player/Custom.lua

```diff
@@ -0,0 +1,98 @@
+---
+-- @Liquipedia
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+local Array = Lua.import('Module:Array')
+local Class = Lua.import('Module:Class')
+local Flags = Lua.import('Module:Flags')
+local Namespace = Lua.import('Module:Namespace')
+local String = Lua.import('Module:StringUtils')
+
+local Injector = Lua.import('Module:Widget/Injector')
+local Player = Lua.import('Module:Infobox/Person')
+local Widgets = Lua.import('Module:Widget/All')
+local Cell = Widgets.Cell
+
+---@class RematchInfoboxPlayer: Person
+local CustomPlayer = Class.new(Player)
+---@class RematchInfoboxPlayerWidgetInjector: WidgetInjector
+---@field caller RematchInfoboxPlayer
+local CustomInjector = Class.new(Injector)
+
+---@param frame Frame
+---@return Html
+function CustomPlayer.run(frame)
+	local player = CustomPlayer(frame)
+	player:setWidgetInjector(CustomInjector(player))
+	return player:createInfobox()
+end
+
+---@param id string
+---@param widgets Widget[]
+---@return Widget[]
+function CustomInjector:parse(id, widgets)
+	local caller = self.caller
+	local args = caller.args
+
+	if id == 'status' then
+		local cells = {}
+		if String.isNotEmpty(args.positions) then
+			local positions = Array.parseCommaSeparatedString(args.positions, ',')
+			local validPositions = Array.map(positions, String.nilIfEmpty)
+			if #validPositions > 0 then
+				local positionsDisplay = Array.map(validPositions, function(pos)
+					local capitalizedPos = String.upperCaseFirst(pos)
+					return '[[:Category:' .. capitalizedPos .. 's|' .. capitalizedPos .. ']]'
+				end)
+				local label = #validPositions > 1 and 'Positions' or 'Position'
+				table.insert(cells, Cell{name = label, content = positionsDisplay})
+			end
+		end
+		return cells
```

Comment:
```suggestion
		local positions = Array.parseCommaSeparatedString(args.positions, ',')
		local validPositions = Array.map(positions, String.nilIfEmpty)
		local label = #validPositions > 1 and 'Positions' or 'Position'
		return {
			Cell{name = label, content = Array.map(validPositions, function(pos)
				local capitalizedPos = String.upperCaseFirst(pos)
				return '[[:Category:' .. capitalizedPos .. 's|' .. capitalizedPos .. ']]'
			end)}
		}
```

agree with rath that standard stuff should be used
but if we keep this could simplify to this